risc0/circuit/rv32im/src/taps.rs

```diff
@@ -16,7 +16,9 @@
 
 use risc0_zkp::taps::{TapData, TapSet};
 
-pub(crate) const TAPSET: &'static TapSet = &TapSet::<'static> {
+#[allow(missing_docs)]
+
+pub const TAPSET: &'static TapSet = &TapSet::<'static> {
```

Comment:
```suggestion
#[allow(missing_docs)]
pub const TAPSET: &'static TapSet = &TapSet::<'static> {
```

Reply:
Unfortunately, this is auto generated so it'll revert back every time we update from private. When I get a little bit of time after this sequence of merges, I can open a PR to remove it from bootstrap